docs/content/3.forms/9.form-group.md

```diff
@@ -132,6 +132,25 @@ You can also use the `error` prop as a boolean to mark the form element as inval
   The `error` prop will automatically set the `color` prop of the form element to `red`.
 ::
 
+### Size
+Use the `size` prop to change the size of the label and the form element.
+
+::component-card
+---
+props:
+  size: 'xl'
+  label: 'Email'
+  hint: 'Optional'
+  description: "We'll only use this for spam."
+  help: 'We will never share your email with anyone else.'
+code: >-
+
+  <UInput size="xl" placeholder="you@example.com" icon="i-heroicons-envelope" />
```

Comment:
You need to **exclud** other props to hide it from preview.

![image](https://github.com/nuxtlabs/ui/assets/37311945/564ab1e4-6926-4221-ace1-4f0b1bb55896)


```md
props:
  size: 'xl'
  label: 'Email'
  hint: 'Optional'
  description: "We'll only use this for spam."
  help: 'We will never share your email with anyone else.'
excludedProps:
  - label
  - hint
  - description
  - help
```

Another thing you need to do is pass the size from `FormGroup` to `Input`

Reply:
I actually want to show it on preview, to demonstrate that all of those get rendered with the size set!

Reply:
Take a look here for [`ButtonGroup`](https://ui.nuxtlabs.com/elements/button#group), which is similar to `FormGroup`. I think it's noisy to show all properties; the main thing you need to focus on is the `size`

Check here how to handle passing `size` from `FormGroup` to children inputs
https://github.com/nuxtlabs/ui/blob/dev/src/runtime/components/elements/ButtonGroup.ts